### PR TITLE
Add audiences to the model to be shown in the Sent Status TM.

### PR DESCRIPTION
**Pull request by tsujonathan**
This PR includes the following change:
- Add audiences to the model to be shown in the Sent Status TM.

### CODE REVIEW

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs [hunk 1]
```diff
@@ -30,5 +31,20 @@ public class SentNotification : BaseNotification
         /// Gets or sets the number of recipients who were throttled out.
         /// </summary>
         public int Throttled { get; set; }
+
+        /// <summary>
+        /// Gets or sets Teams audience id collection.
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs]: Change "id" in this so "... Teams audience name collection"

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs [hunk 2]
```diff
@@ -30,5 +31,20 @@ public class SentNotification : BaseNotification
...
+        /// </summary>
+        public IEnumerable<string> TeamNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets Rosters audience id collection.
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs]: Change "id" in this so "... Rosters audience name collection"

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs [hunk 3]
```diff
@@ -30,5 +31,20 @@ public class SentNotification : BaseNotification
...
+        /// </summary>
+        public IEnumerable<string> RosterNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether a notification should be sent to all the users.
```
Comment (jotrick) [inline Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs]: Change to "... a notification was set to all"

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs (hunk 1 above)
Comment (tsujonathan) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs]: Addressed.

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs (hunk 2 above)
Comment (tsujonathan) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs]: Addressed.

File: Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs (hunk 3 above)
Comment (tsujonathan) [reply Source/Microsoft.Teams.Apps.CompanyCommunicator/Models/SentNotification.cs]: Addressed.

**Pull request merged**